oci/archive/oci_src.go

```diff
@@ -28,6 +28,18 @@ func (e ImageNotFoundError) Error() string {
 	return fmt.Sprintf("no descriptor found for reference %q", e.ref.image)
 }
 
+// ImageStructureNotFoundError occurs when the archive file does not exist.
+type ImageStructureNotFoundError struct {
+	// ref is the image reference
+	ref ociArchiveReference
+	// path is the file path that was not present
+	path string
+}
+
+func (e ImageStructureNotFoundError) Error() string {
+	return fmt.Sprintf("no archive file present for reference %q", e.ref.image)
```

Comment:
something like
```suggestion
	return fmt.Sprintf("archive file %q not found", e.ref.resolvedFile)
```
(optionally with `.file` and/or `.image`, or all of `.StringWithinTransport`) — users are mostly going to experience the error through this text. Not including `resolvedFile` at all could get pretty confusing.